src/window/gui/Gui.cpp

```diff
@@ -337,13 +340,16 @@ void Gui::DrawMenu() {
 
     ImGui::DockSpace(dockId, ImVec2(0.0f, 0.0f), ImGuiDockNodeFlags_None | ImGuiDockNodeFlags_NoDockingInCentralNode);
 
-    if (ImGui::IsKeyPressed(TOGGLE_BTN) ||
+    if (ImGui::IsKeyPressed(TOGGLE_BTN) || ImGui::IsKeyPressed(ImGuiKey_Escape) ||
         (ImGui::IsKeyPressed(TOGGLE_PAD_BTN) && CVarGetInteger(CVAR_IMGUI_CONTROLLER_NAV, 0))) {
-        GetMenuBar()->ToggleVisibility();
+        if (!ImGui::IsKeyPressed(ImGuiKey_Escape)) {
+            GetMenuBar()->ToggleVisibility();
+        }
```

Comment:
it seems with this logic using the controller to toggle the menu will always enable the menu**bar** - is that intended?

Reply:
No, that wasn't intended, actually, but I'm not sure how to make that dynamic instead of just swapping it over.

Reply:
I figured out *a* solution.